server/datastore/mysql/hosts.go

```diff
@@ -27,6 +27,7 @@ import (
 // Since many hosts may have issues, we need to batch the inserts of host issues.
 // This is a variable, so it can be adjusted during unit testing.
 var hostIssuesInsertBatchSize = 10000
```

Comment:
Should we reduce to 5k this one too? (Used by `ds.UpdateHostIssuesVulnerabilities`)